source/manual/emergency-publishing.html.md

```diff
@@ -73,28 +73,23 @@ The data for the emergency banner is stored in Redis. Jenkins is used to set the
 ![Jenkins Deploy Emergency Banner](images/emergency_publishing/deploy_emergency_banner_job.png)
 
 <a name="clear-template-cache"></a>
-### 4. Clear caching in frontend, static and whitehall-frontend
+### 4. Clear caching in frontend apps
 
-1) Run the Fabric task to clear the application template cache for frontend and
-static:
+1) Run the Fabric task to clear the application template cache for Static:
 
 ```
 fab $environment campaigns.clear_cached_templates
 ```
 
-2) Clear the cache for whitehall-frontend and frontend by restarting memcached:
+2) Clear the cache for frontends:
 
-```
-fab $environment class:whitehall_frontend app.restart:memcached
-fab $environment class:frontend app.restart:memcached
-```
-
-> **NOTE:** The main page updates immediately, however whitehall and travel advice can take a couple of minutes before the banner appears.
+- [Clear the frontend caches on integration](https://deploy.integration.publishing.service.gov.uk/job/clear-frontend-caches)
+- [Clear the frontend caches on staging](https://deploy.staging.publishing.service.gov.uk/job/clear-frontend-caches)
```

Comment:
You'll want to have a production link here for when someone does it for real 😱

Reply:
I've not done that to avoid production accidents  - when this gets real people can hopefully come up with the prod URL themselves :)